backend/app/controllers/spree/admin/users_controller.rb

```diff
@@ -131,6 +131,11 @@ def user_params
           attributes += [{ spree_role_ids: [] }]
         end
 
+        # Don't allow users to set another users password.
+        unless can? :update_password, @user
```

Comment:
I'd remove the code comment, it's pretty clear what it does. Also, not sure about this but what about adding this rule in the permission sets, set to true just for documentation? At least in the `Spree::PermissionSets::UserManagement` file?

Reply:
I thought it would be better to only allow admins to update the password, and have people opt into adding `:update_password` permissions. I've updated `UserManagement` to have this permission though to follow the update_email permissions.